trade-finance-manager-ui/scripts/enable-find-utilisation-reports-by-year-dropdown-autocomplete.ts

```diff
@@ -0,0 +1,29 @@
+const enableFindUtilisationReportsByYearDropdownAutocomplete = (): void => {
+  const findUtilisationReportsByYearForm: HTMLFormElement | null = document.querySelector('form#find-utilisation-reports-by-year--form');
+
+  if (!findUtilisationReportsByYearForm) {
+    return;
+  }
+
+  const yearInput: HTMLInputElement | null = document.querySelector('input#find-utilisation-reports-by-year--year-input');
+  if (!yearInput) {
```

Comment:
Can you please add line feed before LOC:9.

Reply:
Could I ask why? I think it reads more clearly if you get the element and have a guard clause immediately after

Reply:
Added